docs/shared/devkit-and-nx-plugins.md

```diff
@@ -42,6 +42,10 @@ executors.
 
 The Nx team maintains a core set of plugins for many application and tooling frameworks. You can write [custom generators](/generators/workspace-generators) and [executors](/executors/creating-custom-builders) for your own workspace. You can also write your own plugin and share it with the community. The [Nx Plugin](/nx-plugin/overview) plugin provides guidance on how you can build your own custom plugins.
 
+### Local workspace plugins
+
+Nx plugins can also be used in the workspace they are generated inside of. For example, if you generate a plugin with `nx g @nrwl/nx-plugin workspace --importPath workspace`, you will be able to immediately make use of generators and executors within it. This would look like `nx g workspace:lib` for generators or `"executor": "workspace:build"` for executors. If you are only planning on making use of generators, Nx provides workspace generators which can be used rather than whole plugins. Generating a full plugin allows you to take advantage of all of the capabilities above, including project inference and graph extension, which is not possible when solely using workspace generators. Additionally, your local plugin could be set as the default collection in `nx.json`, such that running `nx g lib` would call your workspace plugin's lib generator.
```

Comment:
I would have the example be

```bash
nx g @nrwl/nx-plugin myPlugin --importPath @my-org/myPlugin
```

At first I thought that `workspace` was a key word of some kind.

Reply:
Sure, that makes sense. I'll update it.